### PR TITLE
Add description to the README

### PR DESCRIPTION
**Pull request by BerylBogui**
Only the link to the chatbot is missing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,43 @@
+# README
```
Comment (Ndpnt) [inline README.md]: Header not necessary, to remove

File: README.md [hunk 2]
```diff
@@ -1,7 +1,43 @@
...
+
+### [EN]
+
```
Comment (Ndpnt) [inline README.md:4]: Can you add a one-line proposition value?

File: README.md [hunk 3]
```diff
@@ -1,7 +1,43 @@
+# README
+
+### [EN]
```
Comment (Ndpnt) [inline README.md]: Use full language name
## English

File: README.md [hunk 4]
```diff
@@ -1,7 +1,43 @@
...
+
+When fighting disinformation, analyses and fact-checks produced often relay - when they are not based on - **figures to measure the impact of a content** on a given society.
```
Comment (Ndpnt) [inline README.md:5]: Add a header
### Motivation

File: README.md [hunk 5]
```diff
@@ -1,7 +1,43 @@
...
+
+In a spreadsheet (.csv file), the number of engagements (shares, likes and comments alike) generated by an article is linked to: 
+- its region or country of origin,
+- its source (main national newspapers),
+- the date of its publication,
+- its topic (title). 
+
+Therefore, **quantitative data is both contextualized and put into perspective**, in a **clear and accessible** way.
+
+[To the chatbot]()
```
Comment (Ndpnt) [inline README.md]: Remove it while we do not have a link

**Comment by Ndpnt** [comment]
Thanks, I will add technical parts later. And I can handle modifications discussed in my comments if you want.

File: README.md (hunk 2 above)
Comment (BerylBogui) [reply README.md:4]: I don't know what you mean

**Comment by BerylBogui** [comment]
I've taken into account all of your suggestions except this one: "Can you add a one-line proposition value?", which I don't understand.

**Pull request merged**